test/test_errors.py

```diff
@@ -85,10 +85,10 @@ def test_handle_result_error(container_factory, rabbit_config):
             # proxy.task() will hang forever because it generates an error
             proxy.task.call_async()
 
-        with eventlet.Timeout(1):
```

Comment:
Hard to say. I added a timer context manager around this locally and `container.wait` only ever took ~0.005 seconds.